### PR TITLE
Add support for GitHub Actions

### PR DESCRIPTION
**Pull request by bregman-arie**
...

### CODE REVIEW

**Comment by jsanemet** [comment]
This request is really big and contains changes that go from a new source, to more documentation, to modifying utilities... You may want to divide it into smaller chunks so that it is easier to review.

**Comment by bregman-arie** [comment]
> This request is really big and contains changes that go from a new source, to more documentation, to modifying utilities... You may want to divide it into smaller chunks so that it is easier to review.

Yes, it's not ready for review or merge. I found out we have some issues we might want to resolve first in separate PRs. Right now adding a new system requires more changes than it should.

File: cibyl/cli/parser.py [hunk 1]
```diff
@@ -151,4 +151,5 @@ def extend(self, arguments: list, group_name: str, level: int = 0):
                     default=arg.default,
                     level=level)
         except argparse.ArgumentError:
+            import pdb;pdb.set_trace()
```
Comment (cescgina) [inline cibyl/cli/parser.py:154]: in python3 you can use the breakpoint function for this, without needing to import anything. You can even configure to use other debuggers (personally I prefer ipdb) with the following env variable
`export PYTHONBREAKPOINT=ipdb.set_trace`

Comment (bregman-arie) [reply cibyl/cli/parser.py:154]: Thanks. Old habits and VIM shortcuts :)

**Comment by jsanemet** [comment]
At this point, this PR has not received an updated in quite some time. Unless there is interest on it, I would advise in closing it down until further progress.

**Pull request closed without merging**